plugins/org.eclipse.n4js.tester/src/org/eclipse/n4js/tester/TestDiscoveryHelper.java

```diff
@@ -94,7 +95,7 @@
 	@Inject
 	private IN4JSCore n4jsCore;
 	@Inject
-	private QualifiedNameComputer qualifiedNameComputer;
+	private ResourceNameComputer qualifiedNameComputer;
```

Comment:
please update the field name accordingly (please also check in other places)

Reply:
done